### ISSUE TITLE
Figure out Strix A0 vs Strix B0

### ISSUE THREAD

**Issue by makslevental** [nod-ai/iree-amd-aie#769]
Possibly we are using the wrong device model for strix ([here](https://github.com/nod-ai/iree-amd-aie/blob/main/runtime/src/iree-amd-aie/aie_runtime/xaie_hwcfg.c#L175-L178)). This hasn't bitten us because wrt aie-rt they're the same but I would rather be correct and consistent.

**Comment by makslevental** [comment]
It's B0 because the laptops have 8 columns https://github.com/Xilinx/aie-rt/blob/release/main_aig/driver/src/lite/xaie_lite_hwcfg.h#L218-L220. `xrt-smi` I think should be able to tell us the number of columns but I can't get it to work :shrug:.

**Comment by makslevental** [comment]
Yea it's 8 - i put a breakpoint here https://github.com/nod-ai/ROCR-Runtime/blob/22fd9e495baa94c5f957e28116cdff10d37e5c19/runtime/hsa-runtime/core/driver/xdna/amd_xdna_driver.cpp#L117-L121 and got 8 columns.

**Comment by dezhiAmd** [comment]
Run xrt-smi like below:
`xrt-smi validate -r latency gemm --verbose`

On a system with PHX NPU, the output will be:
_Verbose: Enabling Verbosity
Validate Device           : [00c3:00:01.1]
    Platform              : NPU
    Power Mode            : Default
-------------------------------------------------------------------------------
Test 1 [00c3:00:01.1]     : latency
    Description           : Run end-to-end latency test
    Xclbin                : C:\Windows\System32\DriverStore\FileRepository\ipukmddrv.inf_amd64_1a5ca989adaaf7ca\validate_**1502_00**.xclbin_ 

In summary:
phx: 1502_00
stx a0 is 17f0_00
stx b0 is 17f0_10
stx halo is 17f0_11
krk is 17f0_20"

**Comment by dezhiAmd** [comment]
Here is what I found after testing on multiple systems
Run xrt-smi like below:
xrt-smi validate -r latency gemm --verbose
On a system with npu1, the output will be:
Verbose: Enabling Verbosity
Validate Device : [00c3:00:01.1]
Platform : NPU
Power Mode : Default
Test 1 [00c3:00:01.1] : latency
Description : Run end-to-end latency test
Xclbin : C:\Windows\System32\DriverStore\FileRepository\ipukmddrv.inf_amd64_1a5ca989adaaf7ca\validate_1502_00.xclbin
In summary:
npu1: 1502_00
npu2: is 17f0_00
npu4: is 17f0_10
